src/item_pocket.cpp

```diff
@@ -2360,6 +2450,7 @@ void item_pocket::favorite_settings::info( std::vector<iteminfo> &info ) const
         info.emplace_back( "BASE", string_format(
                                _( "Items in this pocket <bad>won't be unloaded</bad> unless you manually drop them." ) ) );
     }
+    info.emplace_back( "BASE", string_format( _( "Preset Name: %s" ), preset_name ) );
```

Comment:
This should be hidden if there is not a preset name. And since the preset name is optional it's better to change its type to `cata::optional<std::string>` to avoid handling the special case of empty string.